sympy/stats/rv.py

```diff
@@ -655,7 +655,15 @@ def given(expr, condition=None, **kwargs):
             if temp == True:
                 return True
             if temp != False:
-                sums += expr.subs(rv, res)
+                # XXX: This seems nonsensical but preserves existing behavoiur
+                # after the change that Relational is no longer a subclass of
+                # Expr. Here expr is sometimes Relational and sometimes Expr
+                # but we are trying to add them with +=. This needs to be
+                # fixed somehow.
```

Comment:
@czgdp1807 do you know what is going on here? This is the `given` function and the `expr` argument is sometimes an `Expr` and sometimes `Relational`. In either case the function attempts to add expr with `sum += expr` which is nonsensical for relations.

Reply:
Apologies, I couldn't respond earlier. May be I didn't receive the automated notification email for this. I hope the problem here got resolved.